src/cpp/session/modules/SessionConnections.R

```diff
@@ -160,26 +160,107 @@ options(connectionObserver = list(
    NULL
 })
 
+.rs.addFunction("connectionReadSnippets", function() {
+   snippetsPath <- getOption("connections-path", "/etc/rstudio/connections/")
```

Comment:
We probably need two settings then, one for users to add their custom connections with the code as it is, but another one for the system admin to add their very own settings. @jmcphers  would about we let them configure a `connections-path` as a RSP profile (http://docs.rstudio.com/ide/server-pro/r-sessions.html#user-and-group-profiles)?